### DISCUSSION TITLE
在开源世界里，应该持有什么样的法律思维 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#61] [Announcements]
# 在开源世界里，应该持有什么样的法律思维 - 「开源之道」

https://opensourceway.community/posts/os-license-and-copyleft/how-to-think-open-source-copyright-risk/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

其实吧, 无法在哪个世界里, 嘦社会主体是人类,
就应该有相同的一个法律思维,
毕竟, 法律本身就是人类社会最核心故事之一,

故事的版本可以有不同,
但是, 故事核心思想/态度/认知/.... 必然是相同的,
否则根本无法协同起来...

而在中国, 开源以及开源许可证 在日常话语体系中的缺席,
其实, 和法律在中国日常中的缺失是相匹配的....

**Reply by lijiangsheng1** [reply]
布道者的无奈～ 写再多的文章，都不如人们吃一次亏的收获大。
